Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/ConvertToWavelength.py

```diff
@@ -0,0 +1,146 @@
+# pylint: disable=too-few-public-methods
+
+""" ConvertToWavelength algorithm converts to wavelength units."""
+
+from __future__ import (absolute_import, division, print_function)
+from mantid.kernel import (Direction, StringListValidator, Property)
+from mantid.dataobjects import EventWorkspace
+from mantid.api import (DataProcessorAlgorithm, MatrixWorkspaceProperty, AlgorithmFactory, PropertyMode, Progress)
+from sans.common.constants import EMPTY_NAME
+from sans.common.general_functions import (create_unmanaged_algorithm, append_to_sans_file_tag,
+                                           get_input_workspace_as_copy_if_not_same_as_output_workspace)
+from sans.common.enums import (RebinType, RangeStepType)
+
+
+class ConvertToWavelength(DataProcessorAlgorithm):
```

Comment:
How SANS specific this algorithm really is? If it is strongly tied to SANS, I suggest renaming it. If not, then the summary and category should be changed as not to scare away potential users.